OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -529,21 +495,28 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
         _slaveBodyForChild = slaveForChild;
     }
 
-private:
-    //=========================================================================
+    //==========================================================================
     // DATA
-    //=========================================================================
+    //==========================================================================
+protected:
+    // Specifies the direction of the Joint in the multibody tree: parent->child
+    // (isReversed is false; default) or child->parent (isReversed is true). The
+    // Joint's transform and coordinates maintain a parent->child sense even if
+    // the joint is reversed.
+    bool isReversed = false;
```

Comment:
> do you think this deserves mention in the changelog?

We haven't been consistent with updating CHANGELOG; we will need to work on it prior to releasing 4.0 (see #384). But yes, I will add to CHANGELOG.

> What about putting this description with the public accessor `isReversed()`? Might be good to also include a note about the history ("Before OpenSim 4.0, the user could set this. In 4.0 and later, this is determined automatically and cannot be set by the user"). I think the reason you put the description here is that this is supposed to be somewhat hidden. Perhaps the accessor's description can say "Advanced."

I am going to remove `getIsReversed()` in part 2 when I improve the tests (`getIsReversed()` is used only by testJoints). The fact that joints can be reversed internally should be *completely* hidden from the user. I will add your note ("Before OpenSim 4.0, …") to CHANGELOG.